suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -195,7 +195,7 @@ export const selectEnabledDiscoveryNetworkSymbols = (state: DiscoveryConfigSlice
 export const selectDeviceEnabledDiscoveryNetworkSymbols = createMemoizedSelector(
     [
         selectDiscoveryNetworkSymbols,
-        selectEnabledDiscoveryNetworkSymbols,
+        selectEnabledDiscoveryNetworkSymbols, // TODO: What about moving the BTC-only logic there?
```

Comment:
If going with this approach, imo putting the logic here is better option as you suggest

Reply:
Also this selector is used for message system so it would make sense to have it here if we want to have btc targeted (and not already enabled by coin enabling)

Reply:
As discussed in person, this was not a good solution for many reasons. Handled in `discoveryMiddleware`.